src/__internal__/checkable-input/checkable-input.component.js

```diff
@@ -40,8 +40,13 @@ const CheckableInput = ({
   warning,
 }) => {
   const { current: id } = useRef(inputId || guid());
-  const labelId = `${id} label`;
-  const helpId = `${id} help`;
+  const labelId = `${id}-label`;
+  const helpId = [error, warning, info].filter(
```

Comment:
Shouldn't `labelHelp` be checked here as well? It will create an icon with Tooltip too

Reply:
Good point, thanks. By the way, I reported a bug for `labelHelp` https://github.com/Sage/carbon/issues/4426